From bc36810e56eaf1ee256a20c37a5ccf05abfa0eb4 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v2-16.27-bc36810e56e-20220323T202132Z-avarab@gmail.com>
In-Reply-To: <cover-v2-00.27-00000000000-20220323T202132Z-avarab@gmail.com>
References: <cover-00.24-00000000000-20220309T123321Z-avarab@gmail.com>
	<cover-v2-00.27-00000000000-20220323T202132Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 30 Sep 2021 18:02:05 +0200
Subject: [PATCH v2 16/27] revisions API: have release_revisions() release
 "commits"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Extend the the release_revisions() function so that it frees the
"commits" in the "struct ref_info".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 revision.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/revision.c b/revision.c
index 90bac9ada03..13b568aea76 100644
--- a/revision.c
+++ b/revision.c
@@ -2937,6 +2937,7 @@ void release_revisions(struct rev_info *revs)
 {
 	if (!revs)
 		return;
+	release_revisions_commit_list(revs);
 	object_array_clear(&revs->pending);
 }
 
-- 
2.35.1.1452.ga7cfc89151f

